Sim_To_DuT_Interface/main.cpp

```diff
@@ -33,20 +33,22 @@
 #include "DuT_Connectors/CANConnector/CANConnectorConfig.h"
 #include "Sim_Communication/SimComHandler.h"
 #include "DuTLogger/DuTLogger.h"
+#include "SystemConfig.h"
 
 int main() {
     DuTLogger::logMessage("Start Application", LOG_LEVEL::INFO);
 
+    // System config
+    sim_interface::SystemConfig systemConfig;
+    sim_interface::SystemConfig::loadFromFile("../../Config/SystemConfig.xml",systemConfig, true);
```

Comment:
Evtl statt relativem Pfad: std::filesystem::canonical("/proc/self/exe").parent_path().string();
gibt dir das aktuelle dir in dem die executable ausgeführt wird.

Reply:
Habe ich umgeändert